### ISSUE TITLE
required UI package like account-ui

### ISSUE THREAD

**Issue by HedCET** [sujith3g/meteor-cordova-google-plus#3]

**Comment by HedCET** [comment]
bootstrap - https://github.com/vcompile/gsignin-bs

**Comment by Alveoli** [comment]
Yep, +1 for this from me of course. Do you know when you might implement this?

**Comment by HedCET** [comment]
@sujith3g @arunhedcet is it possible to merge it's functionality with existing ui packages of meteor ?

**Comment by elie222** [comment]
does this package work with useraccounts?
